File: helloworld/javascript/level1/printTriangle.js
function printTriangle(num) {
    var result = '';
    for (var i = 1; i <= num; i++) {
        result += '*'.repeat(i) + '\n';
    }
    return result
}


// 아래는 테스트로 출력해 보기 위한 코드입니다.
console.log( printTriangle(3) );

File: helloworld/java/level1/PrintTriangle.java
package level1;

public class PrintTriangle {
	public String printTriangle(int num) {
		if(num < 1) {
			return "";
		} else {
			int i = 1;
			
			StringBuffer sb = new StringBuffer();
			
			while(i<=num) {
				for(int j=0;j<i;j++) {
					sb.append("*");
				}
				sb.append("\n");
				i++;
			}
			return sb.toString();
		}
		
	}
	
	public static void main(String[] args) {
		PrintTriangle pt = new PrintTriangle();
		System.out.println(pt.printTriangle(3));
	}
}
